### ISSUE TITLE
Startup error

### ISSUE THREAD

**Issue by emersonveenstra** [emersonveenstra/vbot#45]
```
TypeError: Cannot read property 'match' of undefined
    at Bot.buildContext (/home/vbot/vbot/vbot.js:161:33)
    at Bot.receive_data (/home/vbot/vbot/vbot.js:212:45)
    at emitOne (events.js:77:13)
    at TLSSocket.emit (events.js:166:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at TLSSocket.Readable.push (_stream_readable.js:109:10)
    at TLSWrap.onread (net.js:517:20)
```

probably because https://github.com/emersonveenstra/vbot/blob/master/vbot.js#L161 on joins, message.message is null

**Comment by NinjaBanjo** [comment]
closed with #49

**Comment by NinjaBanjo** [comment]
#49 has been merged
